release-tools/build.make

```diff
@@ -101,9 +106,42 @@ push-%: container-%
 		fi; \
 	done
 
+RELEASE_ALIAS_TAG=$(PULL_BASE_REF)
+
+push-multiarch-%:
+	export DOCKER_CLI_EXPERIMENTAL=enabled
+	make BUILD_PLATFORMS="windows amd64 .exe"
```

Comment:
Why this line?

Reply:
make build for windows is required as Windows binary is needed to copy it in windows dockerfile